features/step_definitions/logging.rb

```diff
@@ -386,3 +387,140 @@
     | component=eventrouter |
   })
 end
+
+Given /^I generate certs for the#{OPT_QUOTED} receiver(?: in the#{OPT_QUOTED} project)?$/ do | receiver_name, project_name |
+  script_file = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/logforwarding/cert_generation.sh"
+  if project_name
+    proj = project_name
+  else
+    proj = "openshift-logging"
+  end
```

Comment:
@QiaolingTang, we can use the ruby shorthand `proejct_name ||= "openshift-logging"` and then use `project_name` instead of the temp variable `proj`